File: problems/1909-remove-one-element-to-make-the-array-strictly-increasing/solution.py
class Solution:
    def canBeIncreasing(self, nums: List[int]) -> bool:
        n = len(nums)
        for i in range(n):
            temp = nums[:i] + nums[i+1:]
            is_increasing = True
            for j in range(1, len(temp)):
                if temp[j] <= temp[j-1]:
                    is_increasing = False
                    break
            if is_increasing:
                return True
        return False

File: problems/1909-remove-one-element-to-make-the-array-strictly-increasing/solution.java
class Solution {
    public boolean canBeIncreasing(int[] nums) {
        for (int i = 0; i < nums.length; i++) {
            int[] temp = new int[nums.length - 1];
            int k = 0;
            for (int j = 0; j < nums.length; j++) {
                if (j != i) {
                    temp[k++] = nums[j];
                }
            }
            boolean increasing = true;
            for (int j = 1; j < temp.length; j++) {
                if (temp[j] <= temp[j - 1]) {
                    increasing = false;
                    break;
                }
            }
            if (increasing) {
                return true;
            }
        }
        return false;
    }
}